Documentation/ApiOverview/Events/Events/Backend/_BeforeRecordDownloadPresetsAreDisplayedEvent/_PresetListener.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyVendor\MyExtension\RecordList\EventListener;
+
+use TYPO3\CMS\Backend\RecordList\DownloadPreset;
+use TYPO3\CMS\Backend\RecordList\Event\BeforeRecordDownloadPresetsAreDisplayedEvent;
+use TYPO3\CMS\Core\Attribute\AsEventListener;
+
+#[AsEventListener(identifier: 'my-package/modify-record-list-preset')]
```

Comment:
If we use `MyExtension` in namespace I would prefer using `my-extension` here. What do you think?

```suggestion
#[AsEventListener(identifier: 'my-extension/modify-record-list-preset')]
```